specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -17,28 +17,11 @@ using Azure.Core.Traits;
 namespace Azure.AI.Language.Conversations;
 
 @doc("Error Response")
-model ErrorResponse {
-    @doc("The error object.")
-    error: Error;
+model ErrorResponse is Azure.Core.Foundations.ErrorResponse{
```

Comment:
I wouldn't even do this. Just reference `Azure.Core.Foundations.ErrorResponse` everywhere or, to make that shorter, make this an `alias` instead.